test/llvm/vectorization/simple_loop_novec.chpl

```diff
@@ -0,0 +1,18 @@
+// This loop shouldn't be vectorized, because
+// LLVM backend cannot check whether A and B overlap
+// And runtime check for overlap was turned off
+// And --vectorize option too
```

Comment:
I think that 'vectorize' should be on by default for the LLVM backend. Therefore the .compots file for this test should have a --no-vectorize.